static/js/src/advantage/distributor/utils/utils.ts

```diff
@@ -70,7 +70,7 @@ export type ChannelProduct = {
   productID: ValidProductID;
   productName: string;
   marketplace: UserSubscriptionMarketplace;
-  version: string;
+  exclusion_group: string;
```

Comment:
For offer types, I considered both normal offers and channel offers. so the exclusion group should be optional for offers type, but for channel offers, I made the exclusion group to always return `""` if it doesn't exist. This also applies to product listings for channel offers, so if the exclusion group is `""` (doesn't exist), it can fetch product listings with exclusion group of `""`